### PR TITLE
Behat refactor

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open still more to go, but this is enough for this PR

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

File: Behat/Page/Page.php [hunk 1]
```diff
@@ -99,4 +92,29 @@ public function getPopover()
...
+        $this->verify($urlParameters);
+
+        return true;
+    }
+
+    protected function verifyUrl(array $urlParameters = array())
+    {
+        $currentUrl = parse_url($this->getDriver()->getCurrentUrl());
+        $expectedUrl = parse_url($this->getUrl($urlParameters));
+        foreach ($expectedUrl as $key => $value) {
```
Comment (chives) [inline Behat/Page/Page.php]: we could try to use https://github.com/rize/UriTemplate instead

**Comment by szymach** [comment]
@fsi-open anything more?

**Pull request merged**